docs/_docs/getting-started/database-support.md

```diff
@@ -160,4 +160,20 @@ java -cp h2-2.1.214.jar org.h2.tools.RunScript \
 [Configuration]: {{ site.baseurl }}{% link _docs/getting-started/configuration.md %}
 [HikariCP]: https://github.com/brettwooldridge/HikariCP
 [Monitoring]: {{ site.baseurl }}{% link _docs/getting-started/monitoring.md %}
-[SQL mode]: https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html
\ No newline at end of file
+[SQL mode]: https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html
+
+### Improve query performance
+
+#### PostgreSQL
+To improve the performance, JPA uses index for faster retrieval. However, some fetch queries while matching strings to make comparison uses String method like LOWER on a field. Index is not applied in such cases.
```

Comment:
Nit: The API server uses JDO, not JPA. Indexes are used by the database itself, not by the Java persistence APIs (JDO, JPA, ...).